src/components/Dialog/SDialog.vue

```diff
@@ -132,6 +132,12 @@ export default class SDialog extends Mixins(BorderRadiusMixin) {
    * `(done: boolean) => {}`
    */
   @Prop({ type: Function }) readonly beforeClose!: (done: boolean) => {}
+  /**
+   * Key for dialog content.
```

Comment:
Please add something like: 'If you need force rerender of table content (for instance, columns were changed), you should set some condition value here'

Reply:
Done